pareto/utilities/get_data.py

```diff
@@ -239,9 +241,46 @@ def get_data(fname, set_list, parameter_list, sum_repeated_indexes=False):
     Similarly, the Set for Water Quality Index "model.s_QC" is derived by the method based
     on the Parameter: PadWaterQuality which is indexed by QC
     """
+    # Using only the names available in the input sheet
+    set_list_common = []
+    parameter_list_common = []
+    df = pd.ExcelFile(fname)
+    sheet_list = df.sheet_names
+    exclusion_list = []
+    for name in sheet_list:
+        if name in set_list:
+            set_list_common.append(name)
+        elif name in parameter_list:
+            parameter_list_common.append(name)
+        else:
+            if name != "Overview" and name != "Schematic":
+                exclusion_list.append(name)
```

Comment:
What is the purpose of appending to this list `exclusion_list`? It seems it isn't actually used for anything.

Reply:
The exclusion list is used to warn the user that the data files have some headers which are not there in the sets and parameter list provided by the user. This is just to make sure that any name changes in the user input should also be incorporated in the data file and vise-versa. That is the reason exclusion list is used.

Reply:
My point is that `exclusion_list` is only referenced in two places in the code - it's initialized with `exclusion_list = []` and then appended to with `exclusion_list.append(name)`. However, it is never used for any other purpose, so you could remove these lines from your code without any affect.

Reply:
Ah ok. I thought I was passing it to the warning but I missed it I was just printing the name of the tab. Thanks, I have removed it. It should be good now.